samples/Shell/Fabimals/Fabimals/App.fs

```diff
@@ -0,0 +1,271 @@
+﻿// Copyright 2018-2019 Fabulous contributors. See LICENSE.md for license.
+namespace Fabimals
+
+open Fabulous.Core
+open Fabulous.DynamicViews
+open Xamarin.Forms
+open System
+open Fabimals.Views
+open Fabimals.Data
+open Fabimals.Models
+open Fabimals.Components
+open Fabimals.Routes
+
+module AppStyles =
+    let applyDefaultShellItemStyle (element: ViewElement) =
+        element.ShellBackgroundColor(Color.FromHex("#455A64"))
+               .ShellForegroundColor(Color.White)
+               .ShellTitleColor(Color.White)
```

Comment:
You define `ShellTitleColor` 2 times.
.ShellTitleColor(Color.White) and .ShellTitleColor(Color.FromHex("#B4FFFFFF"))

I think one time should be enough?

Reply:
Good catch! The 2nd one is a failed copy-paste.

Reply:
This is fixed now